yt_dlp/extractor/callin.py

```diff
@@ -0,0 +1,123 @@
+# coding: utf-8
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    float_or_none,
+    int_or_none
+)
+
+
+class CallinIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?callin\.com/(episode)/(?P<id>[-a-zA-Z]+)'
+    _TESTS = [{
+        'url': 'https://www.callin.com/episode/the-title-ix-regime-and-the-long-march-through-EBfXYSrsjc',
+        'info_dict': {
+            'id': '218b979630a35ead12c6fd096f2996c56c37e4d0dc1f6dc0feada32dcf7b31cd',
+            'title': 'The Title IX Regime and the Long March Through and Beyond the Institutions',
+            'url': 'https://d26nqus0kmgkkk.cloudfront.net/25246512e4b374daea3a23397483c50d241329ff8af4a5bafae02dae06cb0d5f/25246512e4b374daea3a23397483c50d241329ff8af4a5bafae02dae06cb0d5f.m3u8',
```

Comment:
url should not be in test defenition
```suggestion
```

Reply:
Done (in 1c2514ace27278cbfea618cecafe23156262a440)